scripts/installer.py

```diff
@@ -11,6 +11,10 @@
 from time import sleep
 import sys
 
+
+def is_arm():
+    return True if 'arm' in platform.machine().lower() or 'aarch' in platform.machine().lower() else False
+
```

Comment:
`is_arm64()` should be added as well and these should probably just be `==` instead of checking if they contain `aarch`/`arm` because that will match with `aarch64`/`arm64`